docs/installation.rst

```diff
@@ -139,4 +178,25 @@ Usually is useful to install examples along with the XRCE-DDS suite, for doing s
 
 .. note::
     If you want to install the *eProsima Micro XRCE-DDS* examples, you can add :code:`-DUXRCE_BUILD_EXAMPLES=ON`
-    to the :code:`cmake ..` command-line options. This flag will enable the compilation of the examples.
\ No newline at end of file
+    to the :code:`cmake ..` command-line options. This flag will enable the compilation of the examples.
+
+.. _install_agent_client_docker:
+
+Using the provided Docker image
+*******************************
+
+    *eProsima Micro XRCE-DDS* is also available as a whole package in a `Docker image. <https://hub.docker.com/r/eprosima/micro-xrce-dds-agent>`_
```

Comment:
Missing full stop at the end.

Reply:
Same as before, full stop included in the html reference